var/spack/repos/builtin/packages/lua-luajit-openresty/package.py

```diff
@@ -26,7 +26,6 @@ class LuaLuajitOpenresty(LuaImplPackage):
     )
 
     provides("lua-lang@5.1", when="+lualinks")
-    conflicts("lua", when="+lualinks")
```

Comment:
Any idea what these were supposed to mean?

Reply:
I think it's just because `lua` provides `lua-lang` too?